test/expectations/expectations_test_runner.rb

```diff
@@ -3,8 +3,10 @@
 
 class ExpectationsTestRunner < Minitest::Test
   TEST_EXP_DIR = "test/expectations"
-  TEST_FIXTURES_DIR = "test/fixtures"
-  TEST_FIXTURES_GLOB = File.join(TEST_FIXTURES_DIR, "**", "*.rb")
+  TEST_RUBY_LSP_FIXTURES_GLOB = File.join("test/fixtures", "*.rb")
+  TEST_RAILS_FIXTURES_GLOB = File.join("test/fixtures/rails_app", "**", "*.rb")
```

Comment:
Since the fixtures have different extensions, I think we don't need to separate the `rails_app` here from the other ones.